databaseconnect.py

```diff
@@ -169,12 +174,13 @@ def learn_question_response(H):
     cur.execute("SELECT id FROM statement_table ORDER BY id DESC")
     res = cur.fetchone()
     last_id = res[0]
-    cur.execute(f"UPDATE statement_table SET sentence='{H}' WHERE id={last_id}")
+    cur.execute(f"UPDATE statement_table SET sentence={H} WHERE id={last_id}")
```

Comment:
@Pihu1998  As far as I know, the `cur.execute` function can be used to provide variables explicitly as the second parameter for any kind of query, not only `INSERT` queries. I would suggest changing all of these to the form that you've used in those queries. As we've seen from the articles shared by you, a query like this one could be at risk if someone sends the input ID as a query itself.

Reply:
Hi @janakrajchadha , converting the f-strings to normal SQL queries like that of `INSERT` queries is resulting into an error.  What should I do?
Thanks :)

Reply:
Try this
```
cur.execute("UPDATE statement_table SET sentence = '{}' WHERE id = '{}'".format(H, last_id))
```